websites/F/FLO/presence.ts

```diff
@@ -0,0 +1,138 @@
+const presence = new Presence({
+  clientId: "857504781438681089"
+});
+
+type actionType = {
+  actionId?: string;
+};
+
+
+function DataRake(dataset: DOMStringMap) {
+  const _dataset = JSON.parse(JSON.stringify(dataset));
+  return JSON.parse(_dataset.rake);
+}
+
+function getTimestamp(time: string, deletedKey: string = null) {
+  if (deletedKey !== null) time = time.replace(deletedKey, "");
+  time = time.replace("    ", "").replace("\n", "");
+  return Date.parse(`1970 00:${time} GMT+0000`);
+}
+
+function getQuery() {
+  const search = location.search.substring(1);
+  return JSON.parse(
+    `{"${
+      decodeURI(search)
+        .replace(/"/g, '\\"')
+        .replace(/&/g, '","')
+        .replace(/=/g, '":"')
+    }"}`
+  );
+}
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "flo-logo",
+      buttons: [
+        {
+          label: "FLO 웹사이트",
+          url: "https://www.music-flo.com/"
+        }
+      ]
+    }, player = document.querySelector(".playbar_wrap"),
+    playerBar: HTMLInputElement = player.querySelector("input.progress");
+  if (!playerBar.disabled) { // Use first one-time player
+    const playButton: HTMLButtonElement = player.querySelector("button.icon-player"),
+      playButtonActionId: actionType = DataRake(playButton.dataset),
+      title = player.querySelector("p.title").textContent,
+      artist = player.querySelector("p.artist").textContent,
+      allTime = player.querySelector(".time_all").textContent,
+      allHiddenTime = player.querySelector(".time_all").querySelector("span.hidden").textContent,
+      currentTime = player.querySelector(".time_current").textContent,
+      currentHiddenTime = player.querySelector(".time_current").querySelector("span.hidden").textContent,
+      nowDate = Date.now(),
+      allDate = getTimestamp(allTime, allHiddenTime),
+      currentDate = getTimestamp(currentTime, currentHiddenTime);
```

Comment:
Isn't time in unix timestamp or hh:mm:ss format? If yes, use our functions (`presence.getTimestamps()`, `presence.timestampFromFormat()`)